tests/logging_tests/tests.py

```diff
@@ -475,3 +476,23 @@ def test_custom_logging(self):
         out, err = self.run_manage(['check'])
         self.assertNoOutput(err)
         self.assertOutput(out, "System check identified no issues (0 silenced).")
+
+
+class SchemaLoggerTests(SimpleTestCase):
```

Comment:
All the other loggers and logging features (as far as being tested) are handled n `tests.logging_tests` which is why I'm more inclined to keep it there.